packages/kmod-5.15-nvidia/kmod-5.15-nvidia.spec

```diff
@@ -0,0 +1,324 @@
+%global tesla_515 515.65.01
+%global tesla_515_libdir %{_cross_libdir}/nvidia/tesla/%{tesla_515}
+%global tesla_515_bindir %{_cross_libexecdir}/nvidia/tesla/bin/%{tesla_515}
+%global tesla_515_firmwaredir %{_cross_libdir}/firmware/nvidia/%{tesla_515}
```

Comment:
Note for other reviewers: I checked out this branch and then ran:
```
git diff --no-index --word-diff packages/kmod-*-nvidia/kmod-*-nvidia.spec
```

There's a lot of churn in the diff that stems from changing `%{tesla_470}` to `%{tesla_515}`. That's somewhat unavoidable here but in the interests of simplifying future diffs, it might be good to go ahead and rename this macro to `tesla_ver`. That way the next diff will be easier to examine.

Reply:
I can change this for both kmod packages in another PR :+1: